README.md

```diff
@@ -615,6 +615,15 @@ Go to `localhost:8080` to access the login page. Fill the fields with the values
 ![Adminer login](docs/img/adminer_login.png)
 
 
+### Using Elasticvue
+
+An [Elasticvue](https://github.com/cars10/elasticvue) container is also deployed in dev and local mode, allowing users to inspect the Gohan Elasticsearch node in a GUI.
+
+Go to `localhost:8081` to access the Elasticvue interface. Fill the username field with `elastic` and the password field with the value of `BENTOV2_GOHAN_ES_PASSWORD`. The Uri field must use the IP of the gohan-es container on port 9200 (e.g. http://192.168.48.2:9200), it can be found with this command: `docker inspect bentov2-gohan-elasticsearch | grep -i ipaddress`
```

Comment:
add this port (8081) to the above table (`Communicating with services (in development)`)

also reflow text to be maximum 120 chars per line